view/frontend/web/js/model/aslowas.js

```diff
@@ -32,14 +31,15 @@ define([
              */
             process: function(price, options) {
                 self = this;
-                var formatted, priceInt, optionsPrice;
-                formatted = Number(price.replace(/[^0-9\.]+/g,""));
+                var formatted, priceInt, optionsPrice, priceStr;
 
-                if (options.currency_rate) {
-                    formatted = formatted / options.currency_rate;
-                    formatted = formatted.toFixed(2);
+                priceStr = price.replace(/^(-)|[^0-9.,]+/g, '$1').trim();
```

Comment:
move the logic in the helper function and ideally if possible introduce currency for parsing logic to make it more future-proof (future when we deal with a currency that is not multiplied by 100 for the lowest unit, not needed right now though)

Reply:
(Same as below) [This commit](https://github.com/Affirm/Magento2_Affirm/pull/81/commits/4b80b2555afb3d2f4a1b1c72c3a4195c65eb729a) introduces currency parameters to the helper functions.